turbo.json

```diff
@@ -5,44 +5,35 @@
     "build": {
       "dependsOn": ["^build"]
     },
-    "next-auth#build": {
-      "dependsOn": ["^build"],
-      "outputs": [
```

Comment:
This option was added to ensure that there will be a cache hit if the output files are deleted in https://github.com/nextauthjs/next-auth/pull/4025
If we remove this, then there will be a cache hit on running `build` after `clean` on `next-auth` which is not expected 🤔